vividus-tests/src/main/resources/properties/suite/integration/suite.properties

```diff
@@ -33,3 +33,13 @@ ui.visual.ignored-elements=By.id(some_fake_id)
 # and processed by configuration resolver and decryptor
 property=#{2 * 2}
 bdd.variables.global.property-with-int-value=#{2 + 2}
+
+db.connection.testh2db.driver-class-name=org.h2.Driver
+db.connection.testh2db.url=jdbc:h2:mem:test;\
+  INIT=create schema if not exists test\\;\
+  SET SCHEMA test\\;\
+  DROP TABLE IF EXISTS cosmetics\\;\
```

Comment:
is this statement needed?

Reply:
emm... for some reason I have ubuntu-related failed build without this

Reply:
Changed.